src/structures/ReactionCollector.js

```diff
@@ -103,15 +103,15 @@ class ReactionCollector extends Collector {
    */
   dispose(reaction, user) {
     /**
-     * Emitted whenever a reaction is disposed of.
+     * Emitted whenever a reaction is disposed of and the `dispose` option is set to true.
      * @event ReactionCollector#dispose
      * @param {MessageReaction} reaction The reaction that was disposed of
      * @param {User} user The user that removed the reaction
      */
     if (reaction.message.id !== this.message.id) return null;
 
     /**
-     * Emitted whenever a reaction is removed from a message. Will emit on all reaction removals,
+     * Emitted whenever a reaction is removed from a message and the `dispose` option is set to true. Will emit on all reaction removals,
```

Comment:
```suggestion
     * Emitted whenever a reaction is removed from a message and the `dispose` option is set to true. 
     Emits on all reaction removal.
```

this will fix the max line length issue with the linter, and cleans up some wording.